Resources/views/full/feedback_form.html.twig

```diff
@@ -34,61 +19,42 @@
-                        {{ form_widget( form.email, { 'attr': {'class': 'col-md-6'}} ) }}
-                        {{ form_errors( form.email ) }}
-                    </div>
-                </div>
-
-                <div class="control-group {% if form.subject.vars.errors is not empty %}error{% endif %}">
-                    {{ form_label( form.subject, 'Subject'|trans,  { 'attr': {'class': 'control-label'}} ) }}
-                    <div class="controls">
-                        {{ form_widget( form.subject, { 'attr': {'class': 'col-md-6'}} ) }}
-                        {{ form_errors( form.subject ) }}
-                    </div>
-                </div>
-
-                <div class="control-group {% if form.country.vars.errors is not empty %}error{% endif %}">
-                    {{ form_label( form.country, 'Country'|trans,  { 'attr': {'class': 'control-label'}} ) }}
-                    <div class="controls">
-                        {{ form_widget( form.country ) }}
-                        {{ form_errors( form.country ) }}
-                    </div>
-                </div>
-
-                <div class="control-group {% if form.message.vars.errors is not empty %}error{% endif %}">
-                    {{ form_label( form.message, 'Message'|trans,  { 'attr': {'class': 'control-label'}} ) }}
-                    <div class="controls">
-                        {{ form_widget( form.message, { 'attr': {'class': 'col-md-8', 'rows': '10'} } ) }}
-                        {{ form_errors( form.message ) }}
-                    </div>
-                </div>
+                {{ form_start( form ) }}
 
-                {{ form_widget( form.save, { 'label': "Send form"|trans, 'attr': { 'class': 'btn btn-warning pull-right' } } ) }}
+                {{ form_row( form.firstName, {
+                    'label': 'First Name',
+                    'label_attr': { 'class': 'control-label' },
+                    'attr': { 'class': 'col-md-4' }
+                } ) }}
+
+                {{ form_row( form.lastName, {
+                    'label': 'Last Name',
+                    'attr': { 'class': 'col-md-4' }
+                } ) }}
+
+                {{ form_row( form.email, {
+                    'label': 'Email',
+                    'attr': { 'class': 'col-md-6' }
+                } ) }}
+
+                {{ form_row( form.subject, {
+                    'label': 'Subject',
+                    'attr': { 'class': 'col-md-6' }
+                } ) }}
+
+                {{ form_row( form.country, { 'label': 'Country' } ) }}
+
+                {{ form_row( form.message, {
+                    'label': 'Message',
+                    'attr': { 'class': 'col-md-8', 'rows': 10 }
+                } ) }}
+
+                {{ form_row( form.save, {
+                    'label': "Send form",
+                    'attr': { 'class': 'btn btn-warning pull-right' }
+                } ) }}
 
                 {{ form_end( form ) }}
```

Comment:
Watch out here. As you redefine form_row, hidden fields will probably also be affected (e.g. csrf token field)